packages/account/src/Sections/Profile/PhoneVerification/__test__/resend-code-timer.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import ResendCodeTimer from '../resend-code-timer';
+
+describe('ConfirmPhoneNumber', () => {
+    it('should disable button after its clicked', () => {
+        render(<ResendCodeTimer resend_code_text={'Resend code'} />);
+        const resend_button = screen.getByRole('button', { name: 'Resend code' });
+
+        userEvent.click(resend_button);
+
+        expect(resend_button).toBeDisabled();
+    });
+
+    it('should display Resend code if user requests for new code inside email verification section', () => {
```

Comment:
shouldnt we update the description heading?
we are not specifically checking whether its email verification or phone number section inside the component